widgetssdk/src/main/java/com/glia/widgets/chat/adapter/holder/CustomCardViewHolder.java

```diff
@@ -0,0 +1,41 @@
+package com.glia.widgets.chat.adapter.holder;
+
+import android.view.View;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.glia.androidsdk.chat.ChatMessage;
+
+/**
+ * A CustomMessageViewHolder describes an item view
+ * <p>
+ * CustomMessageViewHolder implementation should have fields
+ * for caching potentially expensive View.findViewById(int) results.
```

Comment:
Maybe it makes sense to clarify what fields are meant in "should have fields for caching potentially expensive View.findViewById(int) results"? Unfortunately, I haven't understood completely what is meant by this.

Reply:
Nothing specific for an Android developer :)
All like in `RecyclerView.ViewHolder`: https://developer.android.com/reference/androidx/recyclerview/widget/RecyclerView.ViewHolder

Reply:
Added the example of usage.